web/src/components/tasks/business-formation/business/BusinessStep.test.tsx

```diff
@@ -489,77 +494,121 @@ describe("Formation - BusinessStep", () => {
       });
     }
 
-    describe("Business Designator Options based on Will Practice Law Answer", () => {
-      it.each(corpLegalStructures)(
-        "Shows PA and PC options for Business Designator when Will You Practice Law is Yes",
-        async (legalStructureId) => {
-          await getPageHelper({ businessPersona: "FOREIGN", legalStructureId }, { willPracticeLaw: true });
+    describe("Business Designator", () => {
+      describe("Business Designator secondary label foreign corporation", () => {
```

Comment:
use a Filter over Legal Structures to find all instances where isForeignCorporation === true... Then do a test, where businessPersona: "FOREIGN" and test where businessPersona: "STARTING"